### PR TITLE
Review: Update German translation for Git v2.35.0

### PR DESCRIPTION
**Pull request by ruester**
@ralfth and @PhillipSz: Please have a look at my suggestions for the German translation of Git v2.35.0. Only the commits b63e8aaa6c (fix from @jottkaerr) and d99fa3c4fb needs to be reviewed, the commit 4ab1b1ff6a is the automatically generated one to update the po file.
As always I also updated some other translations that are not part of this Git update. For example omitting the full stop at the end of a sentence if the original string does not contain one or starting with a small letter after a semicolon...

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -1947,83 +1946,83 @@ msgid ""
...
-msgstr "Branch '%s' existiert bereits."
+msgstr "Branch '%s' existiert bereits"
 
 #: branch.c:313
-#, fuzzy, c-format
+#, c-format
 msgid "cannot force update the branch '%s'checked out at '%s'"
-msgstr "Kann Branch '%s' nicht entfernen, ausgecheckt in '%s'."
+msgstr ""
+"kann Aktualisierung des Branch '%s' nicht erzwingen, ausgecheckt in '%s'"
```
Comment (ralfth) [inline po/de.po]: ... des Branches ...

File: po/de.po [hunk 2]
```diff
@@ -4613,9 +4617,9 @@ msgid "Continuing under the assumption that you meant '%s'."
 msgstr "Setze fort unter der Annahme, dass Sie '%s' meinten."
 
 #: help.c:646
-#, fuzzy, c-format
+#, c-format
 msgid "Run '%s' instead [y/N]? "
-msgstr "Stattdessen '%s' ausführen? (y/N)"
+msgstr "Stattdessen '%s' ausführen? (y/N) "
```
Comment (ralfth) [inline po/de.po]: (y/N) should be inside the sentence. Like " ... (y/N)? "

File: po/de.po [hunk 3]
```diff
@@ -10948,14 +10954,14 @@ msgid "Dirty index: cannot apply patches (dirty: %s)"
...
 msgid "Skipping: %.*s"
-msgstr "Wende an: %.*s"
+msgstr "Überspringe: %.*s"
 
 #: builtin/am.c:1832
-#, fuzzy, c-format
+#, c-format
 msgid "Creating an empty commit: %.*s"
-msgstr "redundante, leere Commits behalten"
+msgstr "Leeren Commit erzeugen: %.*s"
```
Comment (ralfth) [inline po/de.po]: "Erzeuge leeren Commit: %.*s"

File: po/de.po [hunk 4]
```diff
@@ -15683,17 +15679,17 @@ msgid "(none)"
 msgstr "(nichts)"
 
 #: builtin/fetch.c:1400
-#, fuzzy, c-format
+#, c-format
 msgid "refusing to fetch into branch '%s' checked out at '%s'"
-msgstr "Kann Branch '%s' nicht entfernen, ausgecheckt in '%s'."
+msgstr "Anfordern der Branch '%s' verweigert, ausgecheckt in '%s'"
```
Comment (ralfth) [inline po/de.po]: "Anfordern in Branch '%s' ..."

File: po/de.po [hunk 5]
```diff
@@ -15854,9 +15844,8 @@ msgid "use <text> as start of message"
 msgstr "<Text> als Beschreibungsanfang verwenden"
 
 #: builtin/fmt-merge-msg.c:28
-#, fuzzy
 msgid "use <name> instead of the real target branch"
-msgstr "<Name> statt 'origin' für Upstream-Repository verwenden"
+msgstr "<Name> statt echter Ziel-Branch verwenden"
```
Comment (ralfth) [inline po/de.po]: <Name> statt echten Ziel-Branch verwenden

File: po/de.po [hunk 6]
```diff
@@ -22766,39 +22749,39 @@ msgid "unable to load existing sparse-checkout patterns"
 msgstr "konnte die existierenden Muster des partiellen Checkouts nicht laden"
 
 #: builtin/sparse-checkout.c:616
-#, fuzzy
 msgid "existing sparse-checkout patterns do not use cone mode"
-msgstr "konnte die existierenden Muster des partiellen Checkouts nicht laden"
+msgstr ""
+"existierenden Muster des partiellen Checkouts benutzen nicht den Cone-Modus"
```
Comment (ralfth) [inline po/de.po:22755]: "existierende Muster..."

File: po/de.po [hunk 7]
```diff
@@ -23081,7 +23061,7 @@ msgstr "behalte Index"
 
 #: builtin/stash.c:1669 builtin/stash.c:1741
 msgid "stash staged changes only"
-msgstr ""
+msgstr "nur Änderungen in den Stash, die zum Commit vorgemerkt wurden"
```
Comment (ralfth) [inline po/de.po]: maybe 'nur Änderungen stashen, die zum Commit vorgemerkt sind'?

File: po/de.po [hunk 8]
```diff
@@ -23346,14 +23326,13 @@ msgid "git submodule--helper sync [--quiet] [--recursive] [<path>]"
...
+#, c-format
 msgid ""
 "Submodule work tree '%s' contains a .git directory. This will be replaced "
 "with a .git file by using absorbgitdirs."
 msgstr ""
-"Arbeitsverzeichnis von Submodul in '%s' enthält ein .git-Verzeichnis\n"
-"(benutzen Sie 'rm -rf', wenn Sie dieses wirklich mitsamt seiner Historie\n"
-"löschen möchten)"
+"Arbeitsverzeichnis von Submodul in '%s' enthält ein .git-Verzeichnis. Durch "
+"Nutzung von \"absorbgitdirs\" wird dieses durch eine .git-Datei ersetzt."
```
Comment (ralfth) [inline po/de.po]: "die Nutzung ..." ?

File: po/de.po [hunk 9]
```diff
@@ -25182,180 +25161,175 @@ msgid "remote-curl: unknown command '%s' from git"
...
 #: contrib/scalar/scalar.c:378
-#, fuzzy
 msgid "when cloning, create full working directory"
-msgstr "konnte aktuelles Arbeitsverzeichnis nicht bekommen"
+msgstr "vollständiges Arbeitsverzeichnis beim Klonen erstellen"
 
 #: contrib/scalar/scalar.c:380
 msgid "only download metadata for the branch that will be checked out"
-msgstr ""
+msgstr "lade nur Metadaten des Branch herunter, der ausgecheckt wird"
```
Comment (ralfth) [inline po/de.po]: "... des Branches ..."

File: po/de.po [hunk 10]
```diff
@@ -25182,180 +25161,175 @@ msgid "remote-curl: unknown command '%s' from git"
...
 msgid "directory '%s' exists already"
-msgstr ""
+msgstr "Verzeichnis '%s' existiert bereits"
 
 #: contrib/scalar/scalar.c:446
-#, fuzzy, c-format
+#, c-format
 msgid "failed to get default branch for '%s'"
-msgstr "Fehler beim Lesen des Standard-Remote-Repositories für Submodul '%s'"
+msgstr "Fehler beim Abfragen des Default-Branch für '%s'"
```
Comment (ralfth) [inline po/de.po]: "... Default-Branches ..."

File: po/de.po [hunk 11]
```diff
@@ -25182,180 +25161,175 @@ msgid "remote-curl: unknown command '%s' from git"
...
-#, fuzzy
 msgid "refusing to delete current working directory"
-msgstr "konnte aktuelles Arbeitsverzeichnis nicht bekommen"
+msgstr "Löschen des aktuellen Arbeitsverzeichnisses wurde verweigert"
 
 #: contrib/scalar/scalar.c:767
-#, fuzzy
 msgid "include Git version"
-msgstr "ungültige Hash-Version"
+msgstr "ungültige Git-Version"
```
Comment (ralfth) [inline po/de.po]: "Git-Version einbeziehen"

**Review by ralfth — CHANGES_REQUESTED** [review]
Thank you for the translation! Please have a look at my suggestions for improvement.

File: po/de.po (hunk 1 above)
Comment (ruester) [reply po/de.po]: thanks! I also fixed the same mistake for the translation in `contrib/scalar/scalar.c:380`

**Comment by PhillipSz** [comment]
Hey, danke für die Übersetzung! Habe gerade keine Zeit für den Review. Aber ich denke @ralfth hat das schon gut gemacht.

Viele Grüße 
Phillip

File: po/de.po (hunk 11 above)
Comment (ruester) [reply po/de.po]: 🙈 thanks!

**Comment by ruester** [comment]
@ralfth Thanks for your review, I applied all your changes 👍

**Comment by ruester** [comment]
There was an update in the upstream repository so I had to rebase to master again and update the de.po

**Comment by ralfth** [comment]
Looks good to me. Thanks!
Reviewed-by: Ralf Thielow <ralf.thielow@gmail.com>

**Comment by ruester** [comment]
https://github.com/git-l10n/git-po/pull/608

**Pull request closed without merging**